Task_Position_the_Cubes/index.css

```diff
@@ -1 +1,46 @@
 /*  Start coding below for POSITION THE CUBES TASK*/
+.parent {
+  width: 300px;
+  height: 300px;
+  /* padding: 80px 50px;
+  padding-bottom: 80px;
+  padding: left 30px;;
+  border: 5px solid gray;
+  border-radius: 0px;*/
+  margin: 100px; 
+  background-color: #e42f68;
+}
+.child-one {
+  background: red;
+width: 100PX;
+height: 100PX;
+margin-left: 200px;
+margin: bottom:0cm;
+} 
+.child-two {
+  background: violet;
+  width: 100PX;
+  height: 100PX;
+  scroll: margin top 100px;
+  
+}
+.child-three{
+  background: yellow;
+width: 100px;
```

Comment:
The width property has a mix of uppercase and lowercase 'PX', it should be lowercase (width: 100px;)